src/ketu/decorators/consumer/protocol.clj

```diff
@@ -0,0 +1,5 @@
+(ns ketu.decorators.consumer.protocol)
+
+(defprotocol ConsumerDecorator
+  (poll! [this consumer-ctx poll-fn] "fn [consumer-context poll-fn] -> Iterable<ConsumerRecord>")
```

Comment:
Could you please add more context to the docstring? I don't understand what it's referring to.

Reply:
consumer-opts is the term used throughout this library, not context

Reply:
consumer-opts is the consumer options passed to the consumer